app/controllers/games_controller.rb

```diff
@@ -31,16 +33,16 @@ def player_passcode
         )
 
         while game.attempts.count < ChancesAndGuesses::CHANCES
-          guess = ValidColor.passcode
+          @guess = MiniMax.new(passcode: passcode_colors).play
```

Comment:
`Minimax#play` returns the definitely guessed answer, instead of a single guess. After this line, the guess is persisted as an `Attempt` value. Either it makes sense to not preserve the guessing, or to preserve each actual guess as an `Attempt`, consider which one's the right one for you.

Reply:
I think I will prefer to preserve each actual guess as an attempt. But as the current game is, it only outputs the correct guess and not preserves the wrong guesses as an attempt.